test/postgresql.autoupdate.test.js

```diff
@@ -578,6 +579,7 @@ describe('autoupdate', function() {
               assert.equal(foreignKeys[0].pkTableName, 'customer_test2');
               assert.equal(foreignKeys[0].fkColumnName, 'customerId');
               assert.equal(foreignKeys[0].fkName, 'fk_ordertest_customerId');
+              assert.equal(foreignKeys[0].onDelete, 'NO ACTION');
```

Comment:
If our auto-discovery is setting `onDelete` to `NO ACTION`, then I think `autoupdate`/`automigrate` must support `NO ACTION` too. Otherwise it's not possible to migrate/update models that were auto-discovered.

Reply:
Make sense, but there's an issue how to distinguish when user explicitly set `onDelete` to `NO ACTION` or when he didn't set anything and PG did it as default ?

For this PR it's not an issue since only `CASCADING` is fully supported but if more rules are supported this will need to be handled.

Is removing this assertion a suitable answer for the scope of this PR ?

Reply:
> Make sense, but there's an issue how to distinguish when user explicitly set `onDelete` to `NO ACTION` or when he didn't set anything and PG did it as default ?

Do we need to distinguish between that? 

> For this PR it's not an issue since only `CASCADING` is fully supported but if more rules are supported this will need to be handled.
> 
> Is removing this assertion a suitable answer for the scope of this PR ?

I am not sure, how is autodiscovery working at the moment? Will it set `onDelete` to `'NO ACTION'` or to `undefined`?

I don't want to expand scope of this PR more than needed, but I feel that we must support migration of models that were defined using the auto-discovery mechanism.

Reply:
Interesting. If I am reading our auto-discovery implementation correctly, then we don't support discovering `onDelete` behavior for foreign keys.

https://github.com/strongloop/loopback-connector-postgresql/blob/ced6c286f7baffaf8f569af747d943d0da96ffa4/lib/discovery.js#L262-L284

If that's the case, then we can close this discussion as resolved.

However, it would be great if you could update README to mention that auto-discovery does not support `onDelete` rules yet.